### PR TITLE
feat(unify): show docs alert after creating spec from component/story in runner

### PR DESCRIPTION
**Pull request by mapsandapps**
- Closes [UNIFY-1170]

### Additional details

My assumption is that this should get dismissed either by clicking the "x" or navigating away. And also that it should only show if you navigate directly to the spec when creating it.

This should work when creating a spec from either the main specs list or the inline specs list.

Screen Shot 2022-03-09 at 10 42 22 AM

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

[UNIFY-1170]: https://cypress-io.atlassian.net/browse/UNIFY-1170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/cypress/e2e/index.cy.ts [hunk 1]
```diff
@@ -647,11 +647,48 @@ describe('App: Index', () => {
           cy.findByRole('dialog', { name: defaultMessages.createSpec.successPage.header }).as('SuccessDialog').within(() => {
             cy.findByRole('link', {
               name: 'Okay, run the spec',
-            }).should('have.attr', 'href', getRunnerHref('src/App.cy.jsx')).click()
+            }).should('have.attr', 'href', getRunnerHref('cypress.config.cy.js')).click()
```
Comment (marktnoonan) [inline packages/app/cypress/e2e/index.cy.ts:650]: This change is in a skipped test, is that intentional?

**Review by marktnoonan — CHANGES_REQUESTED** [review]
LGTM, requesting a one line change as I think the `props: true` might end up confusing somebody down the round. One other tiny comment.

File: packages/app/cypress/e2e/index.cy.ts (hunk 1 above)
Comment (mapsandapps) [reply packages/app/cypress/e2e/index.cy.ts:650]: it's unimportant, but yeah. i had unskipped this test to use it as the base for my other tests, and this change does need to be made. but the skipped test still fails even with this change, so i left it skipped.

Comment (tbiethman) [reply packages/app/cypress/e2e/index.cy.ts:650]: I'm going through all our TODOs/skipped tests right now, so any attempt to make them more runnable is appreciated :)

**Review by tbiethman — APPROVED** [review]
Looks good to me too, I'll reapprove when Mark's requested changes are made.

**Review by marktnoonan — APPROVED** [review]

**Review by tbiethman — APPROVED** [review]

**Pull request merged**